appcues/src/main/java/com/appcues/trait/appcues/BackdropKeyholeTrait.kt

```diff
@@ -92,20 +94,22 @@ internal class BackdropKeyholeTrait(
                 .drawWithContent {
                     drawContent()
 
-                    val shapeCenter = Offset(positionPx.x + sizePx.width / 2, positionPx.y + sizePx.height / 2)
-                    val blurStartPoint = ((sizePx.width / 2) - blurRadiusPx) / (sizePx.width / 2)
-
-                    drawRoundRect(
-                        topLeft = positionPx,
-                        brush = Brush.radialGradient(
-                            colorStops = arrayOf(blurStartPoint to Color.Transparent, 1.0f to Color.Black),
-                            center = shapeCenter,
-                            radius = encompassesRadiusPx.value
-                        ),
-                        size = sizePx,
-                        cornerRadius = rectCornerRadius,
-                        blendMode = BlendMode.DstIn
-                    )
+                    if (sizePx.width > 0 && sizePx.height > 0) {
```

Comment:
is that for when we have the Keyhole trait with no extra configuration? or is that just the size of the rect ? there is probably a better way we could make this work, I'm thinking that even if we have a rect of 0x0, we should be able to draw the keyhole if it has set other settings like blurRadius for instance

Reply:
this would be the case when there is either no target rect, or a target rect with size zero, in theory. Maybe a better way to guard is `encompassesRadiusPx.value` must be > 0.